test/rendering/utils/playwright.ts

```diff
@@ -4,7 +4,7 @@ const playwright = (page: Page) => ({
   async open(url: string) {
     // eslint-disable-next-line playwright/no-networkidle
     await page.goto(url, { waitUntil: "networkidle" });
-    await page.waitForSelector('[data-testid="sn-table"] .sn-table-cell-text');
+    await page.waitForSelector('[data-testid="sn-table"]');
```

Comment:
The selector return a first element with a width of 0.`waitForSelector` would fail because it's think the element is not visible.